phpunit/html/wp-html-tag-processor-standalone-test.php

```diff
@@ -175,7 +175,27 @@ public function test_attributes_parser_treats_slash_as_attribute_separator() {
 	 *
 	 * @covers __toString
 	 */
-	public function test_tostring_applies_the_updates_so_far_and_keeps_the_processor_on_the_current_tag() {
+	public function tostring_returns_updated_html() {
+		$p = new WP_HTML_Tag_Processor( '<hr id="remove" /><div enabled class="test">Test</div><span id="span-id"></span>' );
+		$p->next_tag();
+		$p->remove_attribute( 'id' );
+
+		$p->next_tag();
+		$p->set_attribute( 'id', 'div-id-1' );
+		$p->add_class( 'new_class_1' );
+
+		$this->assertEquals(
+			$p->get_updated_html(),
+			(string) $p
+		);
+	}
```

Comment:
More of nitpicks, but couple things:

1. Docblock @covers seems wrong.
2. What is the exact purpose of this testcase? Does it test something that is not covered by the next testcase?